ddl/util/callback/callback.go

```diff
@@ -58,6 +58,8 @@ type TestDDLCallback struct {
 	OnGetJobBeforeExported  func(string)
 	OnGetJobAfterExported   func(string, *model.Job)
 	OnJobSchemaStateChanged func(int64)
+
```

Comment:
```suggestion
```

Reply:
The line break is used to distinguish plain DDL hooks and reorg hooks.